docs/UserGuide.md

```diff
@@ -384,6 +385,9 @@ Alternatively, you can use csv config files to customize the analysis as before
 * `repo-config.csv`: Add additional rows for the extra repos ([example](repo-config.csv))
 * `author-config.csv`: Add one row for each author in each repo you want to analyze
 
+## Quickstart RepoSense with Netlify
+
+Using Netlify, and a fork of this repo, allows you to quickly get started online and enjoy a real time RepoSense report on your target repository. Further instructions can be found in this [guide](UsingNetlifyGuide.md).
```

Comment:
```diff
- Using Netlify, and a fork of this repo, allows
+ Using Netlify and a fork of this repo allow
```

Prefer explicitly stating requirements, instead of using relative clauses.

Reply:
Aah, I see. Is it some convention? Or is this because of the flow it creates?